File: pat-static/src/main/java/com/rectanglel/patstatic/patterns/stops/StopView.java
package com.rectanglel.patstatic.patterns.stops;

import io.reactivex.subscribers.DisposableSubscriber;

/**
 * <p>View for handling UI things for the getStopRenderRequests on the Port Authority API</p>
 * <p>Created by epicstar on 9/20/16.</p>
 */

public interface StopView<R> {
    DisposableSubscriber<R> stopObserver();
}

File: pat-static/src/main/java/com/rectanglel/patstatic/patterns/stops/StopView.kt
package com.rectanglel.patstatic.patterns.stops

import io.reactivex.subscribers.DisposableSubscriber

/**
 *
 * View for handling UI things for the getStopRenderRequests on the Port Authority API
 *
 * Created by epicstar on 9/20/16.
 */

interface StopView<R> {
    fun stopObserver(): DisposableSubscriber<R>
}
